creator-node/src/services/stateMachineManager/stateReconciliation/index.js

```diff
@@ -59,6 +59,10 @@ class StateReconciliationManager {
       prometheusRegistry
     })
 
+    if (config.get('maxRecurringRequestSyncJobConcurrency') === 0) {
```

Comment:
i don't think we'd ever want to pause this but worth having this in here

Reply:
yep